src/otx/algo/detection/ops/nms.py

```diff
@@ -219,3 +221,220 @@ def batched_nms(
 
     boxes = torch.cat([boxes, scores[:, None]], -1)
     return boxes, keep
+
+
+# Below functions come from mmdeploy
+# https://github.com/open-mmlab/mmdeploy/blob/bc75c9d6c8940aa03d0e1e5b5962bd930478ba77/mmdeploy/mmcv/ops/nms.py
+def multiclass_nms(
+    boxes: Tensor,
+    scores: Tensor,
+    max_output_boxes_per_class: int = 1000,
+    iou_threshold: float = 0.5,
+    score_threshold: float = 0.05,
+    pre_top_k: int = -1,
+    keep_top_k: int = -1,
+    output_index: bool = False,
+) -> tuple[torch.Tensor, torch.Tensor] | tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
+    """Create a dummy onnx::NonMaxSuppression op while exporting to ONNX.
+
+    This function helps exporting to onnx with batch and multiclass NMS op. It
+    only supports class-agnostic detection results. That is, the scores is of
+    shape (N, num_bboxes, num_classes) and the boxes is of shape (N, num_boxes,
+    4).
+    """
+    if version.parse(torch.__version__) < version.parse("1.13.0"):
```

Comment:
We don't need this condition because OTX 2.x supports torch >= 2.0.

Reply:
Nice catch! Removed by [3f62054](https://github.com/openvinotoolkit/training_extensions/pull/3366/commits/3f62054242114aea15660194ea6790d5f68aa851)